modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1033,7 +1033,7 @@
     <string name="settings_auto_download_up_next">Episodes added to Up Next</string>
     <string name="settings_auto_play">Auto play</string>
     <string name="settings_auto_play_summary">When there\'s nothing left to play, automatically pick an episode for me</string>
-    <string name="settings_beta_features">Beta Features</string>
+    <string name="settings_beta_features">Beta features</string>
```

Comment:
I believe this should be lowercase to match the other menu titles and the design spec.